### ISSUE TITLE
Clarify Platform Orientation personas

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#49324]
## User Story
As a Governance team member, I want to understand the types of users who will attend Platform Orientation, so that we can create a user journey that captures the onboarding/orientation process for new VFS team members.

Assignee: @allison0034 @sterkenburgsara 

## Description
The goal of this ticket is to align our team on the characteristics of VFS Team members based on practice area, so that we can create a complete user journeys documenting their Platform orientation experience.

Existing Platform Resources:
[Current personas](https://vfs.atlassian.net/wiki/spaces/AP/pages/2061303831/Who+we+serve)
[More personas](https://vfs.atlassian.net/wiki/spaces/AP/pages/2154496209/Onboarding+and+getting+access+to+tools)
[Previous Platform Orientation Research](https://vfs.atlassian.net/wiki/spaces/SDT/pages/1816985754/Platform+Orientation+Research?NO_SSR=1)
[Platform Orientation - current state service blueprint useful](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1634334965737/bf671d055e1072a001d00cd1d6b7dc89b0987bfc?sender=naomimarcussen8901)

## Impacted Artifacts
* [Provisional personas Mural board](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1669217997162/c5ad978f6948abb51626c817ea4518d2118b0c9c?sender=u792eff8ef82f499a192a4934)

## Tasks

- [ ] Create provisional personas for Platform, Product Managers, Researchers, Designers using available documentation and existing team knowledge.
- [ ] Review front end and back end personas
- [ ] Make any changes to existing personas and add missing information
- [ ] Put all personas in the same visual format using [this Mural board](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1669217997162/c5ad978f6948abb51626c817ea4518d2118b0c9c?sender=u792eff8ef82f499a192a4934)

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] Governance team members have a clear understanding of who engages with Platform Orientation, and the similarities/differences between users across practice areas.

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@allison0034 @sterkenburgsara the tasks you identified seem to only address personas on the VFS side - what about Platform? Would we identify the orientation hosts as a persona?

**Comment by allison0034** [comment]
@shiragoodman just chatted with Sara about this. we are going to include a persona for Platform but we aren't sure if we are going to do the same research as with VFS teams.

**Comment by sterkenburgsara** [comment]
As I dig into this one and start working on the tasks, I feel like developing personas for Platform is not useful to our end goal - I think some sort of alignment criteria, or understanding the differences based on user type and how that relates to orientation is useful (e.g. designers need to know the design system, devs do not), but I think we can possibly build a practice-dependent skills matrix or something instead that helps us understand what touchpoints users will need access to (related to onboarding).

@allison0034 what are your thoughts?

**Comment by shiragoodman** [comment]
I agree. I don't think we need documented personas for Platform, however I do think we need to understand Platform's role - in other words, it's not just Governance team that supports Platform Orientation. Other teams play a part.

**Comment by sterkenburgsara** [comment]
@shiragoodman could you elaborate a little more - how do you see this ticket discovering that information?

**Comment by sterkenburgsara** [comment]
@shiragoodman thanks!

**Comment by sterkenburgsara** [comment]
@shiragoodman this ticket has remained untouched for several sprints. We realized early-on that we needed to conduct interviews to better understand any differences across practice-areas and if those pointed to a need for updated personas on Platform (currently, only a subset of our participants are represented by existing personas). Our research did not reveal significant differences across practice-areas, so we did not feel we had a need or adequate information to update personas. This is not to say that differences do not exist, but our research was so focused on new hires (0-6 months on the job) that commonalities in that shared rose to the top and were the focus of our work. // cc @allison0034
